sections/semantics-forms.include

```diff
@@ -72,12 +76,23 @@
 
   <xmp highlight="html">
     <form>
-      <div><label>Customer name: <input></label></div>
+      <div>
+        <label>
+          Customer name: <input>
+        </label>
+      </div>
       <fieldset>
-        <legend> Pizza Size </legend>
-        <div><label> <input type="radio" name="size"> Small </label></div>
-        <div><label> <input type="radio" name="size"> Medium </label></div>
-        <div><label> <input type="radio" name="size"> Large </label></div>
+        <legend>Pizza Size</legend>
+        <div>
+          <label>
+            <input type="radio" name="size"> Small
+          </label>
+        </div>
+        <div>
+          <label>
+            <input type="radio" name="size"> Large
+          </label>
+        </div>
       </fieldset>
```

Comment:
This looks a lot like a list of options to me, and I would far prefer it to use a list element. (Same goes for a lot of the examples below).